### PR TITLE
net-misc/icaclient: Add myself as maintainer ... also for hdx plugin

### PR DESCRIPTION
**Pull request by henning-schild**

### CODE REVIEW

File: media-plugins/hdx-realtime-media-engine/hdx-realtime-media-engine-2.9.500.2802-r1.ebuild [hunk 1]
```diff
@@ -84,6 +85,10 @@ src_install() {
 
 	insinto "${ICAROOT}"
 	doins HDXRTME.so
+
+	# this is for /opt/Citrix/ICAClient/util/HdxRtcEngine from icaclient
+	# but only used when having the hdx plugin, so we do it here
+	dosym /usr/$(get_libdir)/libunwind.so "${ICAROOT}"/libunwind.so.1
```
Comment (thesamesam) [inline media-plugins/hdx-realtime-media-engine/hdx-realtime-media-engine-2.9.500.2802-r1.ebuild:91]: We can't really do this. There's no guarantee it's actually ABI compatible.

If it needs an ancient libunwind, we need to make a libunwind-compat package.

Comment (henning-schild) [reply media-plugins/hdx-realtime-media-engine/hdx-realtime-media-engine-2.9.500.2802-r1.ebuild:91]: i thought so too, but i can confirm that this trick does work ... the application all of a sudden runs as expected, maybe i would need to symlink the .so.8 to .so.1 to make sure the one without the so version does not change to something which would not work anymore

Comment (henning-schild) [reply media-plugins/hdx-realtime-media-engine/hdx-realtime-media-engine-2.9.500.2802-r1.ebuild]: good news is that ubuntu is already affected and users already discuss that with citrix

https://discussions.citrix.com/topic/416197-hdx-rtme-on-ubuntu-ms-teams-not-optimized/

**Comment by henning-schild** [comment]
there are several open bugs for icaclient, please anyone assign me to all of them

**Comment by ionenwks** [comment]
> there are several open bugs for icaclient, please anyone assign me to all of them

done

**Pull request closed without merging**